### PR TITLE
Update paper_draft.Rmd

### PR DESCRIPTION
**Pull request by LinaMarieD**
Added 2 lines.

### CODE REVIEW

File: paper_draft.Rmd [hunk 1]
```diff
@@ -28,3 +28,13 @@ plot(pressure)
 ```
 
 Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
+
```
Comment (nickhaf) [inline paper_draft.Rmd:33]: Please add a caption with # caption

**Review by nickhaf — CHANGES_REQUESTED** [review]

**Comment by nickhaf** [comment]
Great, thanks. I recommend using more informative commit messages, e.g.: "add python and bash chunk"

**Review by grewered — APPROVED** [review]
awesome, good to know

**Review by nickhaf — APPROVED** [review]
Looks good now, thanks

**Pull request merged**